editor/modes/visual-editor/block-list.js

```diff
@@ -210,7 +210,7 @@ class VisualEditorBlockList extends Component {
 					type="text"
 					readOnly
 					className="editor-visual-editor__placeholder"
-					value={ ! blocks.length ? __( 'Write your story.' ) : __( 'Write…' ) }
+					value={ ! blocks.length ? __( 'Write your story' ) : __( 'Write…' ) }
```

Comment:
Hmmm you prefer with the period? For me that seems so _finite_ especially when you're just about to blog.